### PR TITLE
feat: Serialize config from render to client

### PR DESCRIPTION
**Pull request by mattsjones**
This PR also ensures `loadableReady` has fired before calling client code.

### CODE REVIEW

File: entry/client/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// '__sku_alias__clientEntry' is a webpack alias
+// pointing to the consuming apps client entry
+import client from '__sku_alias__clientEntry';
+
+import { loadableReady } from '@loadable/component';
+
+import configKey from '../configKey';
+
+loadableReady(() => {
+  client(window[configKey] || {});
```
Comment (michaeltaranto) [inline entry/client/index.js]: Should we be deleting the configKey off window after its passed back into client function?

File: entry/client/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { loadableReady } from '@loadable/component';
+
+import clientContextKey from '../clientContextKey';
+
+loadableReady(() => {
+  let clientContext = {};
+
+  if (window[clientContextKey]) {
+    clientContext = window[clientContextKey];
+    delete window[clientContextKey];
```
Comment (markdalgleish) [inline entry/client/index.js]: 👏👏👏

File: entry/render/buildRenderEntry.js [hunk 3]
```diff
@@ -1,21 +1,3 @@
...
-  const app = await render.renderApp({ ...renderParams, SkuProvider });
-
-  return await render.renderDocument({
-    ...renderParams,
-    app,
-    bodyTags: getBodyTags(),
-    headTags: getHeadTags()
-  });
-};
+export default async renderParams => render(renderParams);
```
Comment (michaeltaranto) [inline entry/render/buildRenderEntry.js]: `await render`?

Comment (mattsjones) [reply entry/render/buildRenderEntry.js]: Tbh this is just a passthrough function now so it doesn't even need to be async really.

**Pull request merged**